databases/mdbtools/Portfile

```diff
@@ -18,9 +19,17 @@ categories              databases devel
 maintainers             nomaintainer
 
 depends_lib-append      path:lib/pkgconfig/glib-2.0.pc:glib2
-checksums               rmd160  c7be0c5ee24c7f5ef7f299810c5e7b81ac8c0760 \
-                        sha256  bf4b297a9985e82bc64c8a620adc00e2e3483371a7d280e81249b294fe0e6619 \
-                        size    630798
+checksums               rmd160  2bf781ed7656a4b8209fdb5594bf2ce3b47afa78 \
+                        sha256  6b75aa88cb1dc49ea0144be381c8f14b2ae47c945c895656dbebc155cd9ee14b \
+                        size    630547
+
+platform darwin 8 {
+    patchfiles-append   tiger-gmtime_r.diff
+}
+
+# Thread-local storage is required
+compiler.blacklist-append *gcc-3.* *gcc-4.*
+compiler.blacklist-append {clang < 800}
```

Comment:
ideally setting `compiler.thread_local_storage yes` would be sufficient to select an appropriate compiler; can you try that first please before manually blacklisting compilers?

Reply:
Updated, thanks